app/javascript/utils/react-bootloader.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Bugsnag from '@bugsnag/js'
+import BugsnagPluginReact from '@bugsnag/plugin-react'
+import { ExercismTippy } from '../components/misc/ExercismTippy'
+import { ReactQueryCacheProvider } from 'react-query'
+
+type ErrorBoundaryType = React.ComponentType<any>
+
+type GeneratorFunc = (data: any, elem: HTMLElement) => JSX.Element
+type Mappings = Record<string, GeneratorFunc>
+
+type TurboFrameRenderDetail = {
+  fetchResponse: {
+    response: {
+      headers: {
+        get: (name: string) => string | null
+      }
+    }
+  }
+}
+
+let ErrorBoundary: ErrorBoundaryType = () => <></>
+
+if (process.env.BUGSNAG_API_KEY) {
+  Bugsnag.start({
+    apiKey: process.env.BUGSNAG_API_KEY,
+    releaseStage: process.env.NODE_ENV,
+    plugins: [new BugsnagPluginReact()],
+    enabledReleaseStages: ['production'],
+    collectUserIp: false,
+    onError: function (event) {
+      const tag = document.querySelector<HTMLMetaElement>(
+        'meta[name="user-id"]'
+      )
+
+      if (!tag) {
+        return true
+      }
+
+      event.setUser(tag.content)
+    },
+  })
+  const reactPlugin = Bugsnag.getPlugin('react')
+  if (reactPlugin) {
+    ErrorBoundary = reactPlugin.createErrorBoundary(React)
+  } else {
+    throw new Error('Failed to load Bugsnag\'s react plugin')
+  }
```

Comment:
Previously there was an `else`:

```
else {
  ErrorBoundary = <></>
}
```

Did you remove that on purpose?

Reply:
Where can I find that?

Reply:
loc 23